### DISCUSSION TITLE
Very important software Update

### DISCUSSION THREAD

**Discussion by ihattan2020** [neurodesk/neurodesk.github.io-hugo#449] [General]
Dear NeuroDesk Team,

I would like to have the below software with up-to-date version. Could you please help to get these modules updated?

below is the link for latest versions for Spinalcordtoolbox (v. 6.4)and ANTs (v. 2.5.3)

-https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases

-https://github.com/ANTsX/ANTs/releases 

Many thanks in advance for your help.

Kind Regards,

Ibrahim

**Comment by stebo85** [comment]
Dear @ihattan2020,

yes - that's possible. Can you propose these changes directly through a pull request?

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/spinalcordtoolbox/build.sh

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/ants/build.sh

**Comment by ihattan2020** [comment]
Dear Steffen,

Thank you for your help. I tried to create a pull request, but this note showed up, "Pull request creation failed. Validation failed: must be a collaborator". I merged the request below this request 
https://github.com/NeuroDesk/neurocommand/commit/b664c37ea1cfab121b976a0175e4b5dcc80d05a8

Many thanks in advance for your help.

Kind Regards,

Ibrahim

**Reply by stebo85** [reply]
You need to modify these two files:

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/spinalcordtoolbox/build.sh

https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/ants/build.sh

**Reply by ihattan2020** [reply]
I deleted both links  and still showing same error

**Reply by stebo85** [reply]
you need to change the toolVersion inside these files, not delete them:
<img width="894" alt="image" src="https://github.com/user-attachments/assets/2a5a4536-3a01-48ca-a27f-370ad9003026">

**Reply by ihattan2020** [reply]
Could you  please send it as text?

**Comment by ihattan2020** [comment]
It's a bit of a long process and complicating. Why does not neurodesk keep software up to date ? 

Especially I'm using CAI wks. In the past, it was much easier to solve a such issue by IT team in few hours but now it took over a week after we switched to neurodesk platform?

**Reply by stebo85** [reply]
it would only take a minute if you would update these files - it's just changing the version number and sending a pull request. I have done it now for you. Please send an email to the CAI IT helpdesk to ask for an update of neurodesk on the workstations to pull in these changes.

**Comment by ihattan2020** [comment]
Could you please send me the guide link for pull request steps or an example request to follow?

**Reply by stebo85** [reply]
For next time: Click on this little pen on GitHub:
<img width="1400" alt="image" src="https://github.com/user-attachments/assets/be257d0e-573a-4383-b922-64ce5b34a1c9">

Then change the version number and send a pull request. Done.

**Comment by ihattan2020** [comment]
Their old response, FYI.So,  I'll pass it to them now and hope they will fix this issue.
"Hi Ibrahim,

Our Linux workstations use a mirror copy of Neurodesk. 

We can't update them the latest version. You will need to ask Neurodesk to update their versions.

https://www.neurodesk.org/docs/overview/applications/

Cheers,"

 Thanks once again, Steff.

**Reply by stebo85** [reply]
ok, I will reach out to them directly.

**Comment by stebo85** [comment]
@ihattan2020 - the updates are now installed on the CAI workstations.

**Comment by ihattan2020** [comment]
Thank you very much for your continuing support and help for this amazing Platform.
